### DISCUSSION TITLE
Error on send tx. Any help? 🙏

### DISCUSSION THREAD

**Discussion by Douglasacost** [paritytech/smoldot#3106] [General]
Hello guys, I'm trying to implement this light-client on a webpage (Using the WASM strategy)

Here the RPC method sent 
```{"id":1,"jsonrpc":"2.0","method":"author_submitExtrinsic","params":["0x45028400c6f5bc3043731278e954cd2bf0456a0b7e5b28121af089fdc8353c8fe751255f016009373033f8488f20e970258013886c56c6ed73619820b1ed913ecf32b9db479803a0c524681cb9512dd11b3782eadf73abcdd728017ca0280b779c22bc2a8b2400510900020000dc4b5cae108e10211c6641d7f90dca8ad6f5727e4a307507f3341adffe2f37020700cc829c19"]}```

I got this response
```JSON-RPC response: {"jsonrpc":"2.0","method":"author_extrinsicUpdate","params":{"subscription":"0","result":"Dropped"}}```

This trigger this warning ```[tx-service-para_eden] Failed to validate transaction 0xed38…bf7e: Error in call proof: UnusedProofEntry``` thus the transaction wasn't relayed

I have sent the transaction trough Polkadot.js app and works fine using the same method `author_submitExtrinsic`

Probably I'm missing something, how can I solve it?

Here the implementation file: https://gist.github.com/Douglasacost/24597aaba31473c6903ba27828dfd2f3

**Comment by MamontRed** [comment]
I saw that people wrote about such a problem, but at that time they did not find a solution

**Reply by Douglasacost** [reply]
Hi @MamontRed, That error was a mismatch on the chain, we were using a older version of substrate
